### PR TITLE
add damn on Hydration

### PR DESCRIPTION
**Pull request by caoyu1988**
image
image
image

### CODE REVIEW

**Review by antonkhvorov — APPROVED** [review]

File: chains/v21/chains_dev.json [hunk 1]
```diff
@@ -5911,6 +5911,19 @@
...
                 }
+            },
+            {
+                "assetId": 46,
+                "symbol": "DAMN",
+                "precision": 10,
+                "type": "orml",
+                "icon": "DAMN.svg",
+                "typeExtras": {
+                    "currencyIdScale": "0xf1420f00",
```
Comment (leohar) [inline chains/v21/chains_dev.json:5922]:
```suggestion
                    "currencyIdScale": "0x5e430f00",
```

File: chains/v21/chains_dev.json [hunk 2]
```diff
@@ -5911,6 +5911,19 @@
                     "existentialDeposit": "59084194977843",
                     "transfersEnabled": true
                 }
+            },
+            {
+                "assetId": 46,
+                "symbol": "DAMN",
+                "precision": 10,
```
Comment (leohar) [inline chains/v21/chains_dev.json:5918]:
```suggestion
                "precision": 12,
```

**Comment by blshv** [comment]
@caoyu1988 Hello! I noticed that you provided a PNG image in SVG format. Do you have the original vector/SVG logo by any chance?

**Comment by leohar** [comment]
will close this PR, tokens are added here: https://github.com/novasamatech/nova-utils/pull/3292

**Pull request closed without merging**